packages/azure-kusto-data/example.js

```diff
@@ -11,7 +11,7 @@ const database = "<databaseName>";
 const table = "<tableName>";
 
 const kcs = KustoConnectionStringBuilder.withAadDeviceAuthentication(clusterConectionString);
-const kustoClient = new KustoClient(kcs);
+const kustoClient = new KustoClient(kcs); // After using the client, you should use `close()` to free up resources
```

Comment:
You can add here a real call to close..

Reply:
Where? at the end of the file people would miss it, and here the sample won't work

Reply:
Added to the end too